README.md

```diff
@@ -283,6 +209,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/212622690-046064b7-fe7d-44fc-92e1-2060cc8bfc99.png)

- [ ] Your link to your MIT.md file is broken :fearful: Kindly fix this link to make your readme descriptive :+1: